src/middlewares/validatePaginationData.js

```
@@ -0,0 +1,19 @@
+const { validateRequest } = require("./validateData.js")
+const { getAllUserEventsSchema, getAllEventsSchema } = require("../routes/schemas/events.js")
+
+function selectValidationSchema(req, res, next) {
+  if(req.query.userId){
+    req.query.userId = Number(req.query.userId)
+  }
+  req.query.pageNumber = Number(req.query.pageNumber)
+  req.query.itemsPerPage = Number(req.query.itemsPerPage)
```

implement a test to validate the error that `Joi` should throw when receiving string as values, and it should throw an error, also test the success scenario when we receive numbers